jwst/scripts/tests/test_scripts.py

```diff
@@ -21,9 +21,6 @@
     'jump',
     'linearitycorr',
     'make_header',
-    'migrate_data',
-    'minimum_deps',
```

Comment:
Does anyone know what the entry for `minimum_deps` was for? Never heard of it.

Reply:
That script (used to test the oldest versions of all dependencies in the `oldestdeps` CI job) was moved to: https://github.com/spacetelescope/minimum_dependencies
Removing it from this test makes sense to me.

Reply:
Thanks @braingram . In that case, I approve of this PR.

Reply:
no idea, but it was making the unit test for the scripts fail locally